hello-world/tutorial/index.md

```diff
@@ -0,0 +1,471 @@
+
+```go
+// planet/proto/blog/packet.proto
+// this line is used by starport scaffolding # ibc/packet/proto/message
+// IbcPostPacketData defines a struct for the packet payload
+message IbcPostPacketData {
+	string title = 1;
+    string content = 2;
+    string creator = 3; // < ---
+}
+```
+
+We want to make sure the receiving chain has content on the creator of a blog post. We add this value to our IBC `packet`.
+`SendIbcPost` message automatically has the content of the `sender` of the message, this value is already used and verified as the signer of the message. We can therefore add the `msg.Sender` as creator to the new packet before sending it over IBC
+
+```go
+// planet/x/blog/keeper/msg_server_ibcPost.go
+// Construct the packet
+	var packet types.IbcPostPacketData
+
+	packet.Title = msg.Title
+	packet.Content = msg.Content
+	packet.Creator = msg.Sender // < ---
+	
+	// Transmit the packet
+	err := k.TransmitIbcPostPacket(
+		ctx,
+		packet,
+		msg.Port,
+		msg.ChannelID,
+		clienttypes.ZeroHeight(),
+		msg.TimeoutTimestamp,
+	)
+```
+
+## Receiving the post
+
+We are at the heart of our transaction logic, located inside `planet/x/blog/keeper/ibcPost.go`
+
+This file contain three methods that will help us learn the core methods about IBC packets.
+
+- `TransmitIbcPostPacket` called manually to send the packet over IBC, it can also define the logic before the packet is sent over IBC to another blockchain app
+- `OnRecvIbcPostPacket` hook (automatically called) when a packet is received on the chain, we define here the logic of packet reception
+- `OnAcknowledgementIbcPostPacket` hook when a sent packet is acknowledged on the source chain, we define here logics when we know the packet has been received
+- `OnTimeoutIbcPostPacket` hook when a sent packet is timed out, we define here logics when we know the packet will never be received on the target chain
+
+What we need to add into the source code is the logic inside those functions to add to the datatables accordingly.
+
+On reception of the message, we create a new post with the title and the content on the receiving chain.
+
+<!-- Though we didn't specify it in the command, the post type automatically contains a creator field. This field is filled when the type instance is created through a message. -->
+<!-- We can reuse this field.  -->
+
+In order to identify which blockchain app a message is coming from, but also who created the message, we will use an identifier in the following format:
+
+`<portID>-<channelID>-<creatorAddress>`
+
+These parameters need to be added when creating the IBC packet.
+
+Finally, the AppendPost function generated by starport returns the ID of the new appended post. We can return this value to the source chain through acknowledgment.
+
+Append the type instance on receiving the packet: the `ctx`, our above defined identifier format, the `title` and the `content` as `PostID`
```

Comment:
```suggestion
You must add these parameters when creating the IBC packet.

Finally, the AppendPost function that is generated by Starport returns the ID of the new appended post. You can return this value to the source chain through acknowledgment.

Append the type instance as `PostID` on receiving the packet: (describe what these things are)

- the `ctx`
- the identifier format that you defined earlier
- the `title`
- the `content` 
```

Reply:
I think a bullet list will help clarify here, not sure what these gizmos are, help me